### PR TITLE
Add gold value to Convertibles

### PR DESCRIPTION
**Pull request by vsong**
Implement a script to insert and update gold values for convertible items from Markethunt API, as well as updating Convertibles webpage to display gold/sb value statistics.

### Additional admin steps needed after deployment:

1. Run the following migration script in converter database:
```sql
CREATE TABLE `item_values` (
  `item_id` int NOT NULL,
  `gold_value` bigint DEFAULT NULL,
  `sb_value` double DEFAULT NULL,
  PRIMARY KEY (`item_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
-- set 1 Gold equal to 1 Gold. Do not set to 0, or the MH economy will collapse!
INSERT INTO item_values (item_id, gold_value) VALUES(431, 1);
```
2. Add `insert_gold_values.php` to the daily cron. Markethunt refreshes its data at 12:05am UTC, I recommend running the cron at 12:15am UTC.
3. Copy new config.php variables from config.php.example

### Preview of UI changes:
image

### CODE REVIEW

**Comment by vsong** [comment]
items table migration script:
```
ALTER TABLE mhconverter.items ADD gold_value BIGINT NULL;
ALTER TABLE mhconverter.items ADD sb_value DOUBLE NULL;
```

File: insert_gold_values.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+$daily_data = fetchDailyMarketValuesJson();
+$latest_sb_price = $daily_data[114]['price'];
+
+setPDOConv();
+
+// insert/update gold values for items with markethunt entries
+foreach ($daily_data as $item_id => $item_data) {
+    if ($item_data === NULL) continue;
+
+    $insert_query = $pdo_conv->prepare('INSERT INTO item_values (item_id, gold_value)
```
Comment (DevJackSmith) [inline insert_gold_values.php:39]:
```suggestion
    $insert_query = $pdo_conv->prepare('INSERT INTO item_markethunt (item_id, gold_value) 
```

File: insert_gold_values.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        ON DUPLICATE KEY UPDATE item_id = :item_id2, gold_value = :gold_value2');
+    $insert_query->execute([
+        'item_id' => $item_id,
+        'item_id2' => $item_id,
+        'gold_value' => $item_data['price'],
+        'gold_value2' => $item_data['price']
+    ]);
+}
+
+$query = $pdo_conv->prepare('SELECT item_id, gold_value, sb_value FROM item_values');
```
Comment (DevJackSmith) [inline insert_gold_values.php:50]:
```suggestion
$query = $pdo_conv->prepare('SELECT item_id, gold_value, sb_value FROM item_markethunt');
```

File: insert_gold_values.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+$query->execute();
+
+// update sb values of all items based on their gold values
+while ($row = $query->fetch(PDO::FETCH_ASSOC)) {
+    $item_id = (int)$row['item_id'];
+    $gold_value = $row['gold_value'] === NULL ? NULL : intval($row['gold_value']);
+
+    $update_query = $pdo_conv->prepare('UPDATE item_values
```
Comment (DevJackSmith) [inline insert_gold_values.php:58]:
```suggestion
    $update_query = $pdo_conv->prepare('UPDATE item_markethunt 
```

File: src/searchByItem.php [hunk 4]
```diff
@@ -149,13 +152,16 @@ function getMouseMapsQuery(&$query_all, &$query_one) {
...
+    $query_one = 'SELECT aci.convertible_id as conv, i.name as item, 
         aci.total_convertibles_opened as total,	aci.total_item_quantity as total_items,
         aci.single_convertibles_opened as single_opens, aci.times_with_any,
-        aci.min_item_quantity, aci.max_item_quantity, aci.total_quantity_when_any
+        aci.min_item_quantity, aci.max_item_quantity, aci.total_quantity_when_any,
+        v.gold_value as item_gold_value, v.sb_value as item_sb_value
         from aggr_convertible_item aci
 	        inner join items i
 		        on aci.item_id = i.id
+            left join item_values v
```
Comment (DevJackSmith) [inline src/searchByItem.php:163]:
```suggestion
            left join item_markethunt im
```

File: src/searchByItem.php [hunk 5]
```diff
@@ -149,13 +152,16 @@ function getMouseMapsQuery(&$query_all, &$query_one) {
 
 function getConvertibleQuery(&$query_all, &$query_one) {
     $query_all = 'SELECT c.id, c.name FROM mhconverter.convertibles c ORDER BY c.name ASC';
-    $query_one = 'SELECT aci.convertible_id as conv, i.name as item,
+    $query_one = 'SELECT aci.convertible_id as conv, i.name as item, 
         aci.total_convertibles_opened as total,	aci.total_item_quantity as total_items,
         aci.single_convertibles_opened as single_opens, aci.times_with_any,
-        aci.min_item_quantity, aci.max_item_quantity, aci.total_quantity_when_any
+        aci.min_item_quantity, aci.max_item_quantity, aci.total_quantity_when_any,
+        v.gold_value as item_gold_value, v.sb_value as item_sb_value
```
Comment (DevJackSmith) [inline src/searchByItem.php:159]:
```suggestion
        im.gold_value as item_gold_value, im.sb_value as item_sb_value
```

**Review by DevJackSmith — APPROVED** [review]
lgtm. Changed table name to be a bit more descriptive. Thank you :)

**Comment by vsong** [comment]
Updated PR summary with new table name

**Pull request merged**